hypershift-operator/controllers/scheduler/dedicated_request_serving_nodes_test.go

```diff
@@ -34,6 +34,7 @@ func TestNodeReaper(t *testing.T) {
 			n.Labels[hyperv1.HostedClusterLabel] = fmt.Sprintf("%s-%s", clusterNamespace, name)
 			n.Labels[HostedClusterNamespaceLabel] = clusterNamespace
 			n.Labels[HostedClusterNameLabel] = name
+			n.Labels[HostedClusterLabel] = "true"
```

Comment:
can we try to be more self descriptive here e.g. we could have a HostedClusterScheduledLabel same as the existing HostedClusterScheduledAnnotation = "hypershift.openshift.io/cluster-scheduled"

Also could remind me what's the rationale to only wanting to exclude nodes with scheduled clusters?